src/framework/audio/qml/MuseScore/Audio/VolumePressureMeter.qml

```diff
@@ -33,14 +33,11 @@ Canvas {
     property real minDisplayedVolumePressure: -60.0
     property real maxDisplayedVolumePressure: 0.0
 
-    width: 24
+    width: 6
     height: 140
 
-    renderStrategy: Canvas.Cooperative
-    renderTarget: Canvas.FramebufferObject
-
     QtObject {
-        id: privateProperties
+        id: internals
```

Comment:
usually called `prv` in other places

Reply:
ok, why not

I'll rename it in the next PR